CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ We used to have incremental versioning before `0.1.0`.
 ### Bugfixes
 
 - Fixes ``ImplicitElifViolation`` false positives on a specific edge cases.
+- Fixes I_CONTROL_CODE setting for BadMagicModuleFunctionViolation.
```

Comment:
We use back quotes to highlight markdown. Please, use `I_CONTROL_CODE` and `BadMagicModuleFunctionViolation`